apps/console/src/features/core/utils/route-utils.ts

```diff
@@ -397,12 +397,16 @@ export class RouteUtils {
             },
             {
                 category: monitoring,
-                id: "logs"            
+                id: "logs"
             },
             {
-                category: settings,       
+                category: settings,
                 id: "remoteLogging"
             },
+            {
+                category: settings,
+                id: "smsProviders"
```

Comment:
Let's rethink the placement of this Navbar item.

Reply:
Moved this to branding/notification-channels